src/test/java/mil/dds/anet/test/utils/ResourceUtilsTest.java

```diff
@@ -175,4 +187,23 @@ private void validateHistory(final Object[][] testData, final boolean checkPerso
   private Instant toInstant(final Object testDate) {
     return testDate == null ? null : Instant.parse((String) testDate);
   }
+
+  @Test
+  void shouldContainClassificationEntries() {
+    then(getAllowedClassifications()).isNotNull().isNotEmpty();
+  }
+
+  @Test
+  void shouldPassWithoutExceptionForExistingClassification() {
+    thenNoException().isThrownBy(() -> assertAllowedClassification("undefined"));
```

Comment:
We user the AssertJ `assertThat` style in all our other tests, so in line with that I would prefer:
```java
assertThatCode(() -> assertAllowedClassification("undefined")).doesNotThrowAnyException();
```